src/main/java/hudson/plugin/versioncolumn/JVMConstants.java

```diff
@@ -10,6 +10,8 @@
     public static final int JAVA_7 = 51;
     public static final int JAVA_8 = 52;
     public static final int JAVA_9 = 53;
+    public static final int JAVA_10 = 54;
+    public static final int JAVA_11 = 55;
```

Comment:
I suggest adding Java 12 right away. It's going to land soon, and we will get some early adopters